.vscode/launch.json

```diff
@@ -1,15 +1,26 @@
 {
-  // Use IntelliSense to learn about possible attributes.
-  // Hover to view descriptions of existing attributes.
-  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-  "version": "0.2.0",
-  "configurations": [
-    {
-      "name": "Python: Current File",
-      "type": "python",
-      "request": "launch",
-      "program": "${file}",
-      "console": "integratedTerminal"
-    }
-  ]
-}
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
Should we exclude / ignore `.vscode`?

Reply:
Well yes, but actually no. The GitHub [recommended .gitignore for Visual Studio Code](https://github.com/github/gitignore/blob/master/Global/VisualStudioCode.gitignore) keeps some files in there so that project-wide settings are uniform.